audit/auditLog.json

```diff
@@ -161,6 +161,27 @@
       "auditReportPath": "./audit/reports/2025.01.17_LiFiDexAggregator(v1.4.0).pdf",
       "auditCommitHash": "n/a (one deployed contract instance was audited)"
     },
+    "audit20250219": {
```

Comment:
Why is this PR adding those audit entries?

Reply:
@0xDEnYO I don't understand. This was just audited that's why I added the audit entry?

Reply:
this comment was just based on a commit-based view that showed weird changes. All good.